apps/explorer/src/pages/AddressInfoPage/NFTDetailsModal.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2018 - 2023 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { NFTCollectionUriMetaData } from '@alephium/shared'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import Card from '@/components/Cards/Card'
+import HighlightedHash from '@/components/HighlightedHash'
+import { SimpleLink } from '@/components/Links'
+import LoadingSpinner from '@/components/LoadingSpinner'
+import Modal, { ModalProps } from '@/components/Modal/Modal'
+import Table from '@/components/Table/Table'
+import TableBody from '@/components/Table/TableBody'
+import TableRow from '@/components/Table/TableRow'
+import { deviceBreakPoints } from '@/styles/globalStyles'
+import { NFTMetadata } from '@/types/assets'
+
+interface NFTDetailsModalProps extends Omit<ModalProps, 'children'> {
+  nft?: NFTMetadata
+  collection?: NFTCollectionUriMetaData
+}
+
+const NFTDetailsModal = ({ nft, collection, ...props }: NFTDetailsModalProps) => {
+  const { t } = useTranslation()
+
+  return (
+    <Modal {...props}>
+      {nft ? (
+        <>
+          <Header>
+            <h2>{nft.file?.name}</h2>
+            <HighlightedHash text={nft.id} middleEllipsis maxWidth="200px" textToCopy={nft.id} />
+          </Header>
+          <NFTImageContainer>
+            {nft.file?.image ? <Image src={nft.file?.image} /> : t('Missing image')}
+          </NFTImageContainer>
+
+          <MetadataTablesContainer>
+            {nft.file ? (
+              <>
+                <NFTDetailsContainer>
+                  <Table bodyOnly>
+                    <TableBody>
+                      <TableRow>
+                        <span>{t('Name')}</span>
+                        <span>{nft.file?.name}</span>
+                      </TableRow>
```

Comment:
Same as above comment. I think we should do exactly what you did right below, with the `description`.